integration/ingester_test.go

```diff
@@ -596,12 +595,11 @@ func TestIngesterQueryingWithRequestMinimization(t *testing.T) {
 			defer s.Close()
```

Comment:
Same thing here - streaming is now always enabled, so we can remove the `for _, streamingEnabled := range []bool{true, false} {`

Reply:
Done